### DISCUSSION TITLE
Can Biomes be changed?

### DISCUSSION THREAD

**Discussion by EfectoNexo** [Mojang/minecraft-editor#155] [Feedback]
Hello, I have a question, I would like to know if at some point they will introduce the possibility of changing the world's biomes in the editor.

**Comment by TheMightyDarkness** [comment]
There could also be a /fillbiome command for Bedrock editor.

**Comment by bryhaw** [comment]
Similar to what I posted here: https://github.com/Mojang/minecraft-editor/discussions/210

We unfortunately don't support that in Editor yet! It is something we have discussed and I'm hoping we can prioritize it shortly since we have heard this feedback several times. :)
